src/resolvers/message.ts

```diff
@@ -1,25 +1,76 @@
-import { Message, Resolvers } from '../generated/graphql';
+import { Message, MessagesConnection, Resolvers } from '../generated/graphql';
+import { Op, WhereOptions } from 'sequelize';
 
 import { checkAuth } from '../utils/auth';
+import paginateResults from '../utils/pagination';
 
 const resolver: Resolvers = {
   Query: {
     messages: async (
       _,
-      args, {
-        verifyUser,
-        models,
-      },
-    ): Promise<Message[]> => {
+      args,
+      { verifyUser, models },
+    ): Promise<MessagesConnection> => {
       const { Message: messageModel } = models;
       const auth = verifyUser();
       checkAuth(auth);
 
-      return messageModel.findAll({
-        where: {
-          messageId: 1,
-        },
+      const { message, channelId, pageSize = 20, after } = args;
+      let qryMsg: object = {};
+      let qryAfter: object = {};
+      let qryChannelId: object = {};
+
```

Comment:
I hope you to make clearer variable names.

`qryMsg` ==> `queryMessage`
`qryAfter` ==> 'queryAfter`
...

Also, I hope you to consider if they are necessary once more.